### DISCUSSION TITLE
Error: PrismaClient is not configured to run in Vercel Edge Functions or Edge Middleware

### DISCUSSION THREAD

**Discussion by harshmangalam** [prisma/prisma#24398] [Q&A]
### Bug description

[31m[auth][cause][0m: Error: PrismaClient is not configured to run in Vercel Edge Functions or Edge Middleware. In order to run Prisma Client on edge runtime, either:
- Use Prisma Accelerate: https://pris.ly/d/accelerate
- Use Driver Adapters: https://pris.ly/d/driver-adapters

If this is unexpected, please open an issue: https://pris.ly/prisma-prisma-bug-report

### How to reproduce

https://github.com/harshmangalam/threads.qwik

### Prisma information

--

### Environment & setup

--

### Prisma Version

```
5.14.0
```

**Comment by ludralph** [comment]
Hi @harshmangalam 👋 

Thank you for raising this question.

It appears you are trying to deploy your application to the Edge. You can follow this [documentation](https://www.prisma.io/docs/orm/prisma-client/deployment/edge/deploy-to-vercel) to see how to deploy your application to Vercel Edge.

**Comment by harshmangalam** [comment]
@ludralph I have tried with the provided docs but not got any fixed. I am using Auth.js, Qwikcity and vercel postgres and i am getting error at runtime from  auth-js prisma adapter.
If you want you can look into the repo which is open-sourced
https://github.com/harshmangalam/threads.qwik

**Reply by ludralph** [reply]
Hi @harshmangalam 

Has this issue been resolved? I can see you used Accelerate [here](https://github.com/harshmangalam/threads.qwik/commit/6fa50a2fce454f9a386b5a21e75ce13d70c92c9c#diff-b8f9857fb0b3059ca7b70f2e15d12a40c3a46d1ebaf57b1a0d0b8b6f464b7ba2L16).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
